### PR TITLE
Jwt remote

### PR DESCRIPTION
**Pull request by yuval-k**
added instructions using remote jwks server.

### CODE REVIEW

File: gloo/docs/enterprise/authentication/jwt.md [hunk 1]
```diff
@@ -270,3 +270,148 @@ Alternativly, you can just tear down minikube:
...
+
+
+```json
+{
+  "iss": "solo.io",
+  "sub": "1234567890",
+  "solo.io/company":"solo"
+}
+```
+
```
Comment (ilackarms) [inline gloo/docs/enterprise/authentication/jwt.md]: a screenshot here would be nice if you feel like it

File: gloo/docs/enterprise/authentication/jwt.md [hunk 2]
```diff
@@ -271,9 +271,15 @@ Alternativly, you can just tear down minikube:
...
 
-## Appendix - Use a Remote JWKS Server
+## Appendix - Use a Remote Json Web Key Set (JWKS) Server
 
-Let's demonstrate how to use an external JWKS server with Gloo. Here's the plan:
+In the previous part of the guide we saw how to configure gloo with a public key to verify JWTs.
+In this appendix we will demonstrate how to use an external Json Web Key Set (JWKS) server with Gloo. 
+
+Using a Json Web Key Set (JWKS) server allows us to manage the verification keys independently and 
+centrally. This, for exampl,e can allow for easy key rotation.
```
Comment (ilackarms) [inline gloo/docs/enterprise/authentication/jwt.md]: exampl,e

File: gloo/docs/enterprise/authentication/jwt.md [hunk 3]
```diff
@@ -271,9 +271,15 @@ Alternativly, you can just tear down minikube:
 minikube delete
 ```
 
-## Appendix - Use a Remote JWKS Server
+## Appendix - Use a Remote Json Web Key Set (JWKS) Server
 
-Let's demonstrate how to use an external JWKS server with Gloo. Here's the plan:
+In the previous part of the guide we saw how to configure gloo with a public key to verify JWTs.
```
Comment (ilackarms) [inline gloo/docs/enterprise/authentication/jwt.md]: gloo/Gloo

File: gloo/docs/enterprise/authentication/jwt.md [hunk 4]
```diff
@@ -270,3 +270,161 @@ Alternativly, you can just tear down minikube:
...
+curl localhost:8001/api/v1/namespaces/gloo-system/services/gateway-proxy:80/proxy/api/pets
+```
+
+A request with a token should be accepted:
+```shell
+curl localhost:8001/api/v1/namespaces/gloo-system/services/gateway-proxy:80/proxy/api/pets?access_token=$(cat token.jwt)
+```
+### Conclusion
+We have created a JWKS server, signed a custom JWT and used Gloo to verify that JWT
+and authroize our request.
```
Comment (mitchdraft) [inline gloo/docs/enterprise/authentication/jwt.md]: authorize

File: gloo/docs/enterprise/authentication/jwt.md [hunk 5]
```diff
@@ -270,3 +270,161 @@ Alternativly, you can just tear down minikube:
...
+
+Using a Json Web Key Set (JWKS) server allows us to manage the verification keys independently and 
+centrally. This, for example, can allow for easy key rotation.
+
+Here's the plan:
+
+1. Create a private key (we will use it to sign and verify a custom JWT that we will create). 
+1. Use `openssl` to create the key used to sign the JWT.
+1. We will use `npm` to install a conversion utility to convert the key from PEM to Json Web Key format.
+1. Deploy a JWKS server to serve the key.
```
Comment (mitchdraft) [inline gloo/docs/enterprise/authentication/jwt.md]: JWKs

**Pull request merged**